front-end/src/renderer/pages/Transactions/components/TransactionNodeRow.vue

```diff
@@ -125,6 +132,48 @@ const handleDetails = async () => {
     await redirectToGroupDetails(router, props.node.groupId, 'readyToSign');
   }
 };
+
+/* Functions */
+function checkTruncation() {
+  if (!descriptionRef.value) {
+    return;
+  }
+  isTruncated.value = descriptionRef.value.scrollHeight > descriptionRef.value.clientHeight;
+}
+
+/* Hooks */
+onMounted(() => {
+  nextTick(() => {
+    checkTruncation();
+    if (isTruncated.value) {
+      createTooltips();
+    }
+  });
+  window.addEventListener('resize', checkTruncation);
+});
+
+onUpdated(() => {
+  nextTick(() => {
+    checkTruncation();
+  });
+});
+
+onUnmounted(() => {
+  window.removeEventListener('resize', checkTruncation);
+});
+
+/* Watchers */
+watch(isTruncated, (newValue, oldValue) => {
```

Comment:
As isTruncated is only ever set in checkTruncation(), can we put this logic in checkTruncation and remove the watch? Or is there another reason for the watch?